bigbluebutton-html5/private/locales/en.json

```diff
@@ -25,8 +25,8 @@
   "app.userlist.menu.kickUser.label": "Kick user",
   "app.userlist.menu.muteUserAudio.label": "Mute user",
   "app.userlist.menu.unmuteUserAudio.label": "Unmute user",
-  "app.userlist.useritem.nostatus.arialabel": "{username} {presenter} {you}",
-  "app.userlist.useritem.status.arialabel": "{username} {presenter} {you} current status {status}",
+  "app.userlist.useritem.nostatus.arialabel": "{0} {1} {2}",
+  "app.userlist.useritem.status.arialabel": "{0} {1} {2} current status {3}",
```

Comment:
They're wild cards for actual values. You can't just concatenate them. If the language requires that the values be reversed or positions swapped that can be done in the localization by moving the wild cards around.